### ISSUE TITLE
Support for §

### ISSUE THREAD

**Issue by thueske** [ms-studio/karmilla#42]
Hey guys,

are there any plans for adding the section sign § in Karmilla?

Would be very, very nice :)

Thx.

**Comment by ms-studio** [comment]
That's right, I didn't notice that § is missing from Karla (and Karmilla).

One of the next items on our to-do list!

**Comment by thueske** [comment]
Nice to hear! :)

Thx for your great work!
